NanoCORE/DiPhotonSelections.h

```diff
@@ -74,7 +75,7 @@ struct Photon {
     float isScEtaEE_ = 0;
     //float trkSumPtHollowConeDR03_ = 0; // To be readded
     //int idlevel_ = SS::IDdefault;
-    unsigned char genPartFlav_ = 0;
+    unsigned char genPartFlav_ = -1;
```

Comment:
Why change the default value? There was a reason it was 0 and not something else. More specifically, you can check [here](https://pdg.lbl.gov/2019/reviews/rpp2018-rev-monte-carlo-numbering.pdf) that 0 is the only value not reserved for a particle, with most other values being taken by particles (positive numbers) and their antiparticles (negative numbers). I would change it back to what it was, here and also on L29.

Reply:
Fixed and set back to 0, thanks!